### PR TITLE
Changed Shared Fonts section to not overstate browser cache disadvantages

### PR DESCRIPTION
**Pull request by jakeleichtling**
@mkruisselbrink / @jyasskin PTAL

### CODE REVIEW

File: use-cases.html [hunk 1]
```diff
@@ -141,16 +141,16 @@
...
-          automatically as soon as the cache becomes stale, this load performance hit is
-          experienced whenever the font changes. Additionally, when the cache is stale, the entire
-          font must be refreshed, as opposed to efficiently downloading the deltas.
+          Having a way to preload and persist fonts locally can provide a much better user
+          experience for CJK (and also Latin) users. The browser cache provides a rudimentary
+          persistence mechanism, allowing a font drawn from a given URL to be retrieved from the
+          server once and served locally many times. However, the browser cache falls short of the
+          ideal implementation. For example, when a font is changed, the browser cache does not
+          realize it is stale until a web page next requests the font. At this point, the browser
+          cache immediately hits the server to pull down the new font. In an ideal world, this
```
Comment (jyasskin) [inline use-cases.html]: Maybe add

> The [`stale-while-revalidate` Cache-Control extension](http://tools.ietf.org/html/rfc5861#section-3) can allow the web page to use the font without a delay, but it still causes the new font to be downloaded while the user is actively using the network for other purposes.

**Comment by jyasskin** [comment]
1 comment, then LGTM.

**Comment by jakeleichtling** [comment]
@jyasskin Should I link to the stale-while-revalidate [RFC](http://tools.ietf.org/html/rfc5861#section-3) (or some other place) in the section it is mentioned and the glossary?

**Comment by jyasskin** [comment]
I would (you'll note that my comment included the link), but I tend to link much more aggressively than most people.

**Pull request merged**